libs/langchain-community/jest.config.cjs

```diff
@@ -5,6 +5,7 @@ module.exports = {
   modulePathIgnorePatterns: ["dist/", "docs/"],
   moduleNameMapper: {
     "^(\\.{1,2}/.*)\\.js$": "$1",
+    "^@utils/(.*)$": "<rootDir>/src/utils/$1",
```

Comment:
What is this for?

Reply:
Coming back to this now. This was added by "yarn build" at the top level - when I remove it the build adds it back in. I'm really not sufficiently familiar with this whole build environment/config to tell you why it is being added?